website/client/components/chat/chatCard.vue

```diff
@@ -234,18 +189,6 @@ export default {
       }
       return likeCount;
     },
-    tierIcon () {
```

Comment:
(not related to this PR in particular, but since in another one this function is used as well, can it be moved to a mixin? I think the existing mixins/styleHelper would be perfect)